couchbase-persistence/README.md

```diff
@@ -1,42 +1,15 @@
-Couchbase Lagom Samples
-=======================
+# Couchbase Lagom Samples
 
 These are simple examples demonstrating how to use Couchbase with Lagom both in Java with Maven and in Scala with Sbt.
 These sample apps use PersistentEntity with Couchbase as a backend. Also they implement simple read-side processors that 
 consume events and update a couchbase document that can be queried.
 
 > Note: Couchbase specific parts are marked with `#couchbase-begin` and `#couchbase-end` in multiple files.
 
+These recipes assume you are already familiar with Lagom and particularly with the [Lagom Hello World example app](https://www.lagomframework.com/documentation/1.4.x/java/UnderstandHello.html) in which these recipes are based on.
 
-What it can do
---------------
 
-1) Get user's greeting from the write-side
-
-```
-$ curl http://localhost:9000/api/hello/Alice
-
-Hello, Alice!✔ 
-```
-
-2) Change Alice's greeting
-
-```
-$ curl -H "Content-Type: application/json" -X POST -d '{"message": "Hi"}' http://localhost:9000/api/hello/Alice
-
-{ "done" : true }✔ 
-```
-
-3) Get all users with their current greetings from the read-side
-
-```
-$ curl http://localhost:9000/api/user-greetings
-
-[{"user":"Alice","message":"Hi"}]✔ 
-```
-
-How to run
-----------
+## How to run
 
 In order to run these samples you'll need a Couchbase service. It also requires to pre-create necessary indexes.
```

Comment:
Would it make sense to copy the Docker Compose files and instructions into this project? It would make it easier to run without cloning multiple repos and jumping between different documentation. Disregard if we're still expecting it to change a lot and fall out of sync.

Reply:
If we do that we should wait until we have a final 1.0 just in case there are any more index changes (those go in the docker setup)

Reply:
Sounds good. No need to block this merge on that.